### PR TITLE
Country.ts

### PR DESCRIPTION
**Pull request by donbasuno**
Adding country as example since it's used in various objects proposed by the FO program.

### CODE REVIEW

**Comment by espenove** [comment]
I suggest removing space and hyphens in the ID's, and can you remove $ from $type, inline with the other discussion?

File: src/examples/Country.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+interface Country extends Context {
+  type: 'fdc3.country',
+  name?: string;
```
Comment (rikoe) [inline src/examples/Country.ts]: The optional name property is already inherited from the base context type. You can include it for completeness if you like, but it is redundant.

Comment (espenove) [reply src/examples/Country.ts]: for a country, you will need a name, so the ? should be removed

Comment (rikoe) [reply src/examples/Country.ts]: Agreed, the country is not much use without a name.

**Comment by espenove** [comment]
Resolved in https://github.com/FDC3/ContextData/pull/15

**Pull request closed without merging**